x-pack/plugins/apm/public/components/app/RumDashboard/ImpactfulMetrics/JSErrors.tsx

```diff
@@ -133,6 +134,13 @@ export function JSErrors() {
       <EuiBasicTable
         data-test-subj={'uxJsErrorTable'}
         loading={status !== 'success'}
```

Comment:
Shouldn't this be:
```suggestion
        loading={status === FETCH_STATUS.SUCCESS}
```
?

Reply:
Yeah, I missed that, and also another one up there.